nginx/datadog_checks/nginx/nginx.py

```diff
@@ -281,7 +281,7 @@ def _get_plus_api_data(self, endpoint, nest):
             r = self._perform_request(url)
             payload = self._nest_payload(nest, r.json())
         except Exception as e:
-            if not self.only_query_enabled_endpoints and endpoint in PLUS_API_STREAM_ENDPOINTS.values():
+            if not self.only_query_enabled_endpoints and endpoint in list(PLUS_API_STREAM_ENDPOINTS.values())[0]:
```

Comment:
Good catch! Can this also include the endpoints in the other API versions?  https://github.com/DataDog/integrations-core/blob/master/nginx/datadog_checks/nginx/const.py#L32

Reply:
Good catch! I pushed before I commited. The newly pushed changes should have this.

Reply:
Nice! Since there's some more logic now, what do you think about making this a helper function ie `stream_endpoints_list()` and also writing a test for it?

Reply:
Good idea. Moved it and created a unit test for it.